include/dlaf/eigensolver/bt_band_to_tridiag/mc.h

```diff
@@ -135,8 +138,103 @@ auto updateE(pika::threads::thread_priority priority, WSender&& tile_w, W2Sender
       pika::execution::experimental::start_detached();
 }
 
+struct Helper {
+  Helper(const SizeType b, const SizeType nrefls, matrix::Distribution dist,
+         const GlobalElementIndex offset)
+      : nrefls_(nrefls), input_spec_{dist.tileElementIndex(offset),
+                                     {std::min(b, dist.size().rows() - offset.row()),
+                                      std::min(b, dist.size().cols() - offset.col())}} {
+    const TileElementIndex& sub_offset = input_spec_.origin;
+
+    const SizeType tile_row = dist.globalTileFromGlobalElement<Coord::Row>(offset.row());
+    const bool isLastRow = tile_row == dist.nrTiles().rows() - 1;
+
+    const SizeType max_size = dist.size().rows() - offset.row() - 1;
+    if (isLastRow && max_size < 2 * b - 1) {
```

Comment:
If it is the last row, `max_size` should be <= `b`,  so the second part of the condition has effect only for b=1.
Is this correct?